.github/workflows/go.yaml

```diff
@@ -8,13 +8,16 @@ on:
       - master
 jobs:
   build:
-    name: Build and Lint
+    name: Build and Lint ${{ matrix.go }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.16' ]
```

Comment:
I don't think this has the intended effect. The root tests basically never change and shouldn't be a problem with 1.11.

However, some modules in this repo use a Go version > 1.11. We shouldn't try to build those with 1.11, obviously.

So, I think this needs to be done in Kokoro. Why didn't the upgrade PR break when we brought in a dependency that depended on Go > 1.11? It did for a different dependency (x/net).

Reply:
I'll go ahead and drop this PR.